paper.md

```diff
@@ -433,10 +442,20 @@ We present a few case studies using an ML workflow to demonstrate how Popper hel
 Next, we compare Popper with existing state-of-the-art workflow engines illustrating its YAML based workflow syntax that has a relatively low entry barrier and its ability to run containerized workflows without requiring access to any cloud environment.
 
 As future work, we have planned to add support for more container engines like NVIDIA Pyxis, Charliecloud, Shifter and resource managers like HTCondor, TORQUE to Popper in order to extend the range of the different computing environments currently supported.
-We also plan to add a compatibility layer between the Popper syntax and more advanced workflow languages such as CWL/WDL to enable interoperability between different workflow engines.
-This would also allow Popper users to migrate to other workflow engines without making any changes to their previously written workflows, thus keeping the overhead minimal.
-Several other features have also been planned like workflow report generation, GUI dashboard, etc. to make the tool more user friendly.
+We plan to add more exporter plugins for exporting Popper workflows to advanced workflow syntaxes such as CWL, WDL, and Airflow to enable interoperability between different workflow engines.
```

Comment:
these is mixing bullets with narrative, so it needs to pick either of the two. I think we can make a bullet out of the exporting stuff, and then another one for abstracting builders.